<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherWidget</class>
 <widget class="QWidget" name="LauncherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Startup Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textStatus">
     <property name="text">
      <string>Status: Initializing...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="__verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="textVersion">
     <property name="text">
      <string>Version: Unknown</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
